roles/defaults/vars/main.yml

```diff
@@ -92,8 +92,8 @@ _atmosphere_images:
   kube_proxy: registry.k8s.io/kube-proxy:v1.22.17
   kube_scheduler: registry.k8s.io/kube-scheduler:v1.22.17
   kube_state_metrics: registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.9.2
-  kubectl: docker.io/bitnami/kubectl@sha256:bd420268ae3424b3ab3174e26b895fd8dc464589a8cd62654b9aa739d00ff280 # image-source: docker.io/bitnami/kubectl:latest
-  libvirt: quay.io/vexxhost/libvirtd@sha256:d400204e0332dc815827e5902038a1c672446c58633ba97ede9e20f8ae9a2349 # image-source: quay.io/vexxhost/libvirtd:yoga-focal
+  kubectl: docker.io/bitnami/kubectl:1.27.3
+  libvirt: quay.io/vexxhost/libvirtd@sha256:4c6ce66f2b957eaf18d35c8236a038bd2289c78eec7c21addfa357b30f97b4ed # image-source: quay.io/vexxhost/libvirtd:zed-jammy
```

Comment:
as we don't build focal for zed anymore, I think we are save to directly using zed instead of `zed-jammy` :)